.github/workflows/cf-deploy-test.yml

```diff
@@ -14,8 +14,10 @@ jobs:
       - uses: actions/setup-node@v3
         with:
           cache: yarn
-          node-version: "18"
-      - run: yarn install --frozen-lockfile
+          node-version: 18
+      - run: corepack enable
+      - run: yarn set version stable
```

Comment:
```suggestion
```

don't do this, this might install new version which will have different lockfile. Corepack ensures version from package.json is installed